components/CollectiveNavbar.js

```diff
@@ -39,12 +46,14 @@ const MainContainer = styled.div`
 const MenuLink = styled.a`
   display: block;
   color: #71757a;
-  font-size: 16px;
-  line-height: 24px;
-  letter-spacing: -0.2px;
+  font-size: 14px;
+  line-height: 16px;
   text-decoration: none;
   white-space: nowrap;
-  padding: 12px 16px 16px;
+
+  letter-spacing: 0.6px;
+  text-transform: uppercase;
+  font-weight: 200;
```

Comment:
I see `500` in https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC.com%5D-Production-Ready-%E2%9C%85?node-id=6564%3A1849

Reply:
True but it looked incorrect (too chunky) at `500`. I can put it back though.

Reply:
We should try to match the design, if it doesn't look good we'll catch during demo & iterate :-)